CRM/Core/Form.php

```diff
@@ -1292,7 +1292,7 @@ public function addDateRange($name, $from = '_from', $to = '_to', $label = 'From
    *
    * Return string
    */
-  private function getApiAction() {
+  protected function getApiAction() {
```

Comment:
changing visibility can break forms that extend the parent - I did a grep & found a few places that implement function getApiAction() with 'protection' but none with private so I guess that is fine. Also it's a bit intuitive that changing up from private is probably ok since private is not available to override